### PR TITLE
Sprint1

### PR DESCRIPTION
**Pull request by jhenry09**

### CODE REVIEW

File: doc/sprint1.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  ### Actionable Commitments
+  
+  * Group needs to decide on library to use in animation of the data structures 
+  * Group should continue to think of which data structures and algorithms to include 
+  
+  ## Planning
+  
+  Points | Story
+  -------|--------
+  5      | User wants to be able to see animation of linked list.
```
Comment (zkhussain) [inline doc/sprint1.md:51]: Should this start with: As a user, ...

Comment (jhenry09) [reply doc/sprint1.md:51]: I think so, I just looked on mountain goat software and his use stories do. Will change

Comment (zkhussain) [reply doc/sprint1.md]: Oh ok, at the course website, I saw it start with that. So thought I would mention it.

**Review by zkhussain — APPROVED** [review]
lgtm

**Review by jhurst502 — APPROVED** [review]

**Pull request merged**